### PR TITLE
[Flow] Upgrade to 68

### PR DESCRIPTION
**Pull request by jasonLaster**
Fixes Issue: #5687

### CODE REVIEW

File: src/reducers/sources.js [hunk 1]
```diff
@@ -126,12 +126,12 @@ function update(
...
     case "ADD_TAB":
       return state.merge({
-        tabs: updateTabList({ sources: state }, action.url)
+        tabs: updateTabList(state.sources.tabs, action.url)
       });
 
     case "MOVE_TAB":
       return state.merge({
-        tabs: updateTabList({ sources: state }, action.url, action.tabIndex)
+        tabs: updateTabList(state.sources.tabs, action.url, action.tabIndex)
```
Comment (darkwing) [inline src/reducers/sources.js]: Would it be easier if we did `state.sources.tabs || []`?

Comment (jasonLaster) [reply src/reducers/sources.js]: hmm, what's the benefit of that?

Comment (darkwing) [reply src/reducers/sources.js]: I (thought I) saw a circle failure due to `updateTabList` returning undefined

Comment (jasonLaster) [reply src/reducers/sources.js]: got it. We might be able to add a sensible default in initializer

**Comment by wldcordeiro** [comment]
👀 looking at this and then the 70 PR now.

**Pull request merged**